src/core/js/views/blockView.js

```diff
@@ -1,16 +1,22 @@
 define([
     'core/js/views/adaptView'
-    ], function(AdaptView) {
+], function(AdaptView) {
 
     var BlockView = AdaptView.extend({
 
+        attributes: function() {
```

Comment:
I thought you had. But alas it's the articleView. Perhaps it would be better in one place if you're able and it works?